### PR TITLE
Publish updates from master

### PR DESCRIPTION
**Pull request by usha-mandya**
Publish the latest changes from master

### CODE REVIEW

File: security/index.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+The Couchbase Server Community Edition is not impacted by this vulnerability, as
+this product does not contain the Couchbase Analytics service. Only Couchbase
+Server Enterprise Edition, when running the Couchbase Analytics service,
+versions 6.0.0 through 6.6.3 and versions 7.0.0 through 7.0.2 are impacted.
+
+For more information, refer to the [Couchbase
+blog](https://blog.couchbase.com/what-to-know-about-the-log4j-vulnerability-cve-2021-44228/){:
+target="_blank" rel="noopener" class="_"}.
+
```
Comment (justincormack) [inline security/index.md:97]: ok, so this is a mitigation if you dont upgrade. We should not recommend this, and it is not clear how these instrucxtions could be applied in a container, and applying them is harder than upgrading, so delete the section.

File: security/index.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+#### Elasticsearch
+
+Set the JVM option `-Dlog4j2.formatMsgNoLookups=true`. For more information,
+refer to the [ElasticSearch
+blog](https://discuss.elastic.co/t/apache-log4j2-remote-code-execution-rce-vulnerability-cve-2021-44228-esa-2021-31/291476){:
+target="_blank" rel="noopener" class="_"}
```
Comment (justincormack) [inline security/index.md:103]: Unclear when you should do this, the page says they have mitigated so i think we should delete this.

File: security/index.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+#### Flink
+
+Set the environment variable `env.java.opts: -Dlog4j2.formatMsgNoLookups=true`
```
Comment (justincormack) [inline security/index.md:107]: remove this line

File: security/index.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+Flink versions `1.11.5`, `1.12.6`, `1.15.0`, `1.14.1`, `1.13.4` have been
+updated to use Apache 2.15.0. Flink `1.14.1` is expected to be released in the
```
Comment (justincormack) [inline security/index.md:110]: you say 14.1 is both released and expected to be released. Which is correct?

None of these versions are on Docker Hub https://hub.docker.com/_/flink so looks like this is not updated yet.

File: security/index.md (hunk 3 above)
Comment (justincormack) [inline security/index.md:107]: this is not the environemnt variable. This is no longer a recommended mitigation. So remove.

File: security/index.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+```console
+dbms.jvm.additional=-Dlog4j2.formatMsgNoLookups=truedbms.jvm.additional=-Dlog4j2.disable.jmx=true
+```
+
+For more information, refer to the [Neo4j announcement](https://community.neo4j.com/t/log4j-cve-mitigation-for-neo4j/48856){:
+target="_blank" rel="noopener" class="_"}.
+
+#### Solr
+
+Manually update the version of log4j2 on your runtime classpath and restart your
```
Comment (justincormack) [inline security/index.md:127]: This is not advice for people using containers, please remove.

**Comment by usha-mandya** [comment]
Discussed with Justin. This is fixed in https://github.com/docker/docker.github.io/pull/13988

**Review by justincormack — APPROVED** [review]

**Pull request merged**